bloom/commands/git/import_upstream.py

```diff
@@ -205,6 +206,12 @@ def handle_tree(tree, directory, root_path, version):
                 file_data = show(BLOOM_CONFIG_BRANCH, os.path.join(root_path, rel_path))
             # Write file
             with open(rel_path, 'wb') as f:
+                # Python 2 will treat this as an ascii string but
+                # Python 3 will not re-decode a utf-8 string.
+                try:
+                    file_data = file_data.encode('utf-8')
+                except UnicodeDecodeError:
+                    file_data = file_data.decode('utf-8').encode('utf-8')
```

Comment:
decoding just to re-encode seems less than ideal but maybe it's fine. Is there another way to indicate to python2 when a str is already unicode. With out the explicit `decode('utf-8')` before re-encoding it assumes the string is ascii and fails.